rust/datafusion/src/physical_plan/hash_aggregate.rs

```diff
@@ -882,25 +867,12 @@ mod tests {
         assert_eq!(batch.num_columns(), 2);
         assert_eq!(batch.num_rows(), 3);
 
-        let a = batch
-            .column(0)
-            .as_any()
-            .downcast_ref::<UInt32Array>()
-            .unwrap();
-        let b = batch
-            .column(1)
-            .as_any()
-            .downcast_ref::<Float64Array>()
-            .unwrap();
-
-        assert_eq!(*a, UInt32Array::from(vec![2, 3, 4]));
+        let mut rows = crate::test::format_batch(&batch);
+        rows.sort();
+
         assert_eq!(
-            *b,
-            Float64Array::from(vec![
-                1.0,
-                (2.0 + 3.0 + 2.0) / 3.0,
-                (3.0 + 4.0 + 4.0) / 3.0
-            ])
+            rows,
```

Comment:
Could we keep the expressions? `(2.0 + 3.0 + 2.0) / 3.0` is much more informative than `2.3333333333333335`, imo.

Reply:
The tests changed to be string comparisons so I am not sure we can keep the arithmetic anymore. 

Maybe we could put it in comments, something like

```
            vec![
                    "2,1.0", 
                    "3,2.3333333333333335",  // 3, (2 + 3 + 2) / 3
                    "4,3.6666666666666665" // 4, (3 + 4 + 4) / 3
            ]
```

Reply:
Makes sense, added the comments